README.md

```diff
@@ -111,22 +112,43 @@ Specify the output directory. The default option is the current directory.
 
 ##### --prettyPrint / -p
 
-Boolean flag to specify if prettyprint the JSON output. The default option is `false`.
+If `-p` flag is specified, the JSON output conforms to prettyprint. The default option is `false`.
 
+##### --elementary
+
+If `--elementary` flag is specified, the CXF (jsonld) files for the elementary blocks are also generated. Else, they are ignored. The default option is `false`. 
+`-o`/`--output` should be `cxf`.
+
+##### --cxfCore
+
+If `--cxfCore` flag is specified, generate the  CXF-core.jsonld files for all the elementary blocks. The default option is `false`.  
+`-o`/`--output` should be `cxf`, `-f`/`--file` should be `path/to/CDL` and `--elementary` flag must be used.
 
 ## 4. JSON Schemas
 
-The JSON representation of Modelica and CDL models must be compliant with the corresponding JSON Schema. This is applicable for the JSON output, not for the raw-json one.
+The JSON representation of Modelica and CXF models must be compliant with the corresponding JSON Schema. This is applicable for the JSON and CXF output respectively.
 
 JSON Schemas describe the data format and file structure to ensure the quality of the JSON files.
 
 Two schemas are available (links to the raw files) :
-- [schema-cdl.json](schema-cdl.json) validates the JSON files parsed from CDL
+- [schema-cxf.json](schema-cxf.json) validates the JSON files parsed from CDL classes to form CXF representations
 - [schema-modelica.json](schema-modelica.json) validates the JSON files parsed from Modelica models
 
 Graphical viewers are available (please use right click + open in a new tab or refresh the page if necessary - this is not optimized for Firefox) :
-- [CDL Schema viewer](cdl-viz.html)
-- [Modelica Schema viewer](modelica-viz.html)
+- [CXF Schema viewer](https://htmlpreview.github.io/?https://github.com/lbl-srg/modelica-json/blob/issue214_cxf/cxf-viz.html)
+- [Modelica Schema viewer](https://htmlpreview.github.io/?https://github.com/lbl-srg/modelica-json/blob/master/modelica-viz.html)
+
+## 5. CXF-Core.jsonld
+
+[CXF-Core.jsonld](CXF-Core.jsonld) contains the CXF representation of all CDL elementary blocks, classes and relationships: It can be used to validate a CXF sequence. 
```

Comment:
how can the CXF representation of the CDL elementary blocks be used to validate the CXF sequences?

Reply:
We can use the CXF-Core.jsonld to make sure that the CXF sequence aligns with the standard (make sure they inputs and outputs of elementary blocks used make sense etc.)

Reply:
But maybe I'll remove it now - we can add it once we have a validation tool.